pinot-core/src/main/java/org/apache/pinot/core/segment/creator/impl/geospatial/H3IndexCreator.java

```diff
@@ -172,6 +175,7 @@ public void seal()
     //write header file
     headerStream.writeInt(VERSION);
     headerStream.writeInt(writer.getNumUniqueIds());
+    headerStream.writeShort(_resolution.size());
```

Comment:
why size, we should write the serialized INT right?

Reply:
Good catch